### ISSUE TITLE
Old replies have no u-in-reply-to class

### ISSUE THREAD

**Issue by unrelentingtech** [ckruse/wwwtech.de#179]
On a recent mention https://wwwtech.de/notes/347 everything is fine:

```html
<a class="u-in-reply-to" href="https://unrelenting.technology/replies/2017-05-28-11-58-07" rel="in-reply-to">https://unrelenting.technology/replies/2017-05-28-11-58-07</a>
```

But on older ones like https://wwwtech.de/notes/146 the class is missing:

```html
<a class="" href="https://unrelenting.technology/articles/freebsd-on-the-thinkpad-x240" rel="in-reply-to">https://unrelenting.technology/articles/freebsd-on-the-thinkpad-x240</a>
```

I wonder how that bug happened… ;)

**Comment by ckruse** [comment]
Hehe, nice. That's because I added a `note_type` column and set the default to `'note'`. And based on this column I set the class:

~~~elixir
  def link_class_by_type("reply"), do: "u-in-reply-to"
  def link_class_by_type("repost"), do: "u-repost-of"
  def link_class_by_type(_), do: ""
~~~

Fixing this needs either some manual tweaking of the data or a date-based switch. Thank for bringing this to my attention!

**Comment by ckruse** [comment]
Fixed this in 1.0.9. Sorry that it took so long, I was overwhelmed with work
